tests/test_classifier/test_rocauc.py

```diff
@@ -224,10 +278,12 @@ def test_rocauc_no_curves(self):
         Test ROCAUC with no curves specified at all
         """
         # Create and fit the visualizer
+        visualizer = ROCAUC(LogisticRegression(), per_class=False, macro=False, micro=False)
+        visualizer.fit(self.binary.X.train, self.binary.y.train)
+
+        # Attempt to score the visualizer
         with pytest.raises(YellowbrickValueError, match="no curves will be drawn"):
-            ROCAUC(
-                LogisticRegression(), per_class=False, macro=False, micro=False
-            )
+            visualizer.score(self.binary.X.test, self.binary.y.test)
```

Comment:
Should we create a test that checks that an exception is raised when `self._binary_decision is True` and `micro`, `macro`, and `per_class` are all also set to `True`?

Reply:
Great idea - I actually just added three to check for each possible combo, just in case.